Remove expo-payments-stripe (#14382)
expo/expo: docs/pages/versions/v41.0.0/sdk/payments.md

import InstallSection from '~/components/plugins/InstallSection';
import PlatformsSection from '~/components/plugins/PlatformsSection';

> > **This library is deprecated in favor of [`@stripe/stripe-react-native`](./stripe.md), and will no longer be available after the release of SDK 43.** [Here's a guide to help make the transition as easy as possible](https://github.com/expo/fyi/blob/master/payments-migration-guide.md#how-to-migrate-from-expo-payments-stripe-to-the-new-stripestripe-react-native-library).

Reviewer: ```suggestion
> **This library is deprecated in favor of [`@stripe/stripe-react-native`](./stripe.md), and will no longer be available after the release of SDK 43.** [Learn more about how to transition to the new library](https://github.com/expo/fyi/blob/master/payments-migration-guide.md#how-to-migrate-from-expo-payments-stripe-to-the-new-stripestripe-react-native-library).
```

reads a bit better to me
